README.md

```diff
@@ -1,14 +1,14 @@
 <img width="600" src="https://rawgit.com/Leaflet/Leaflet/main/src/images/logo.svg" alt="Leaflet" />
 
 Leaflet is the leading open-source JavaScript library for **mobile-friendly interactive maps**.
-Weighing just about 39 KB of gzipped JS plus 4 KB of gzipped CSS code, it has all the mapping [features][] most developers ever need.
+Weighing in at about 39 KB of gzipped JS plus 4 KB of gzipped CSS code, it has all the mapping [features][] most developers ever need.
 
-Leaflet is designed with *simplicity*, *performance* and *usability* in mind.
+Leaflet was designed with *simplicity*, *performance* and *usability* in mind.

```

Comment:
Why use past tense here? This makes it sound like Leaflet is no longer actively being worked on.